examples/otel-demo/otel-demo-values.yaml

```diff
@@ -0,0 +1,322 @@
+# official schema for otel demo is at https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/main/charts/opentelemetry-demo/values.yaml
+
+# Disable all bundled infrastructure components
+opentelemetry-collector:
+  enabled: false
+
+jaeger:
+  enabled: false
+
+prometheus:
+  enabled: false
+
+grafana:
+  enabled: false
+
+opensearch:
+  enabled: false
+
+# Global configuration pointing to Jaeger
+default:
+  env:
+    - name: OTEL_SERVICE_NAME
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: "metadata.labels['app.kubernetes.io/component']"
+    # Override default collector to point to Jaeger
+    - name: OTEL_COLLECTOR_NAME
+      value: jaeger-collector.jaeger.svc.cluster.local
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://jaeger-collector.jaeger.svc.cluster.local:4318
+    - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+      value: http://jaeger-collector.jaeger.svc.cluster.local:4318/v1/traces
+    - name: OTEL_EXPORTER_OTLP_PROTOCOL
+      value: http/protobuf
+    - name: OTEL_RESOURCE_ATTRIBUTES
+      value: service.namespace=otel-demo,deployment.environment=minikube-demo
+    # Disable logs and metrics export to Jaeger (only traces)
+    - name: OTEL_LOGS_EXPORTER
+      value: none
+    - name: OTEL_METRICS_EXPORTER
+      value: none
+    - name: OTEL_TRACES_EXPORTER
+      value: otlp
+
+# Component-specific configurations using correct schema names
```

Comment:
Why do we need to redefine all of this? It's going to be very fragile - if upstream demo changes our settings will become stale.